web/public/locales/en.json

```diff
@@ -176,6 +176,10 @@
       "body": "The data consists of an aggregation of hourly values. {{percentage}}% of the production values are estimated."
     }
   },
+  "units": {
+    "g-kwh": "g / kWh",
```

Comment:
I think this is where the unit is defined so commenting here.

It should be CO2eq / kWh to match the tooltip here:
<img width="604" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/a7860f1d-ae8c-4f0a-bc65-a950374f843f">

Reply:
I would also not add these as translations but rather as constants instead as these should not change between languages as they are base SI units, prefixes and chemical compositions.

Reply:
Okay, I just used `g / kWh` because that was the unit next to the graph before, but I agree, it makes more sense to use `gCO2eq / kWh`. I'll move it out of the translation files.